### PR TITLE
Update k8s fed docs to clarify role of acl token

### PR DESCRIPTION
**Pull request by lkysow**
* also fix some formatting. Will be easier to review if you hide whitespace changes.

### CODE REVIEW

File: website/content/docs/k8s/installation/multi-cluster/vms-and-kubernetes.mdx [hunk 1]
```diff
@@ -108,33 +119,41 @@ setting:
...
 e7924dd1-dc3f-f644-da54-81a73ba0a178
 ```
 
 This token will be used in the server config for the replication token.
-You must also create your own agent policy and token.
 
 ```hcl
 acls {
   tokens {
-    agent = "<your agent token>"
```
Comment (lkysow) [inline website/content/docs/k8s/installation/multi-cluster/vms-and-kubernetes.mdx:124]: removed this here

File: website/content/docs/k8s/installation/multi-cluster/vms-and-kubernetes.mdx [hunk 2]
```diff
@@ -108,33 +119,41 @@ setting:
...
     replication = "e7924dd1-dc3f-f644-da54-81a73ba0a178"
   }
 }
 ```
 
-1. If gossip encryption is enabled, you'll need the key as well. The command
-   to retrieve the key will depend on which Kubernetes secret you've stored it in.
+-> **NOTE:** You'll also need to set up additional ACL tokens as needed by the
```
Comment (lkysow) [inline website/content/docs/k8s/installation/multi-cluster/vms-and-kubernetes.mdx:141]: and added this here

**Review by dnephin — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/368473.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 83a657929913e22601924807744642b68a3efa3f onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 83a657929913e22601924807744642b68a3efa3f onto `release/1.10.x` succeeded!

**Pull request merged**